docs/API.md

```diff
@@ -93,6 +93,9 @@ import { AuthUIContextProvider } from '@brightlayer-ui/react-native-auth-workflo
 -   **showSelfRegistration**: _`boolean`_
     -   When true, shows the Create Account button to allow for self registration.
     -   Default: true
+-   **showLoginForm**: (optional) _`boolean`_
+    -   When true, the Login Form will be part of Login Screen
```

Comment:
```suggestion
    -   When true, the login form will be part of the Login screen.
```

Reply:
I will also add here why anyone would want to use this. Like "This will be useful when paired with xxxx to do xxxxx"